tests/testProfileChangePassword.php

```diff
@@ -20,7 +20,7 @@ class TestProfileChangePassword extends WP_Auth0_Test_Case {
 
 	use UsersHelper;
 
-	use httpHelpers;
+	use HttpHelpers;
```

Comment:
You're changing the case of the class name but the file name is still the same

Reply:
File name is fine, will get picked up by the autoloader. Testing individual files failed when the case of the trait didn't match the case of the `use` statement.